detection_rules/kbwrap.py

```diff
@@ -230,6 +265,39 @@ def kibana_export_rules(
 
             exceptions.append(exception)
 
+    # Parse action connector results from API return
+    action_connectors = []
+    if export_action_connectors:
+        action_results, _ = parse_action_connector_results_from_api(action_corrector_results)
+
+        # Build TOMLException Objects
+        for connector in action_results:
```

Comment:
any reason why its named `connector` here but `actions_dict` in the function definition?

Reply:
Only to match the `def from_exceptions_dict` that I did in exceptions, will update connector so it matches.